website/docs/r/storage_share.html.markdown

```diff
@@ -89,9 +93,9 @@ A `access_policy` block supports the following:
 
 ~> **Note:** Permission order is strict at the service side, and permissions need to be listed in the order above.
 
-* `start` - (Optional) The time at which this Access Policy should be valid from, in [ISO8601](https://en.wikipedia.org/wiki/ISO_8601) format.
+* `start` - (Optional) The time at which this Access Policy should be valid from. When using `storage_account_id` this should be in RFC3339 format. If using the deprecated `storage_account_name` property, this uses the [ISO8601](https://en.wikipedia.org/wiki/ISO_8601) format.
 
-* `expiry` - (Optional) The time at which this Access Policy should be valid until, in [ISO8601](https://en.wikipedia.org/wiki/ISO_8601) format.
+* `expiry` - (Optional) The time at which this Access Policy should be valid untilWhen using `storage_account_id` this should be in RFC3339 format. If using the deprecated `storage_account_name` property, this uses the [ISO8601](https://en.wikipedia.org/wiki/ISO_8601) format.
```

Comment:
Will we have broken this behaviour since we added `RFC3339` validation onto those properties in the resource?

Reply:
RFC3339 and ISO8601 are compatible in the Go `time` package and both pass the validation function. Tests are passing locally for both value formats.